README.md

```diff
@@ -28,19 +28,30 @@ Contributions are very welcome.
 
 Suggested flow:
 
- - Open an issue to discuss the fix / improvement. Might not be needed for trivial things.
- - Fork the repo and create a feature/bug-fix branch from `dev` branch. For trivial changes you might want to work directly on forked `dev` branch.
+ - Open an issue to discuss the fix / improvement. Might not be needed for 
+ trivial things.
+ - Fork the repo and create a feature/bug-fix branch from `dev` branch. 
+ For trivial changes you might want to work directly on forked `dev` branch.
  - Send a PR to `dev` branch
 
+You will find the full description on top of the [issues](issues). There are 
```

Comment:
Instead of linking just to Issues in GitHub, consider linking directly to:
```
[CONTRIBUTING.md](.github/CONTRIBUTING.md)
```

That way, if this README is read outside of GitHub, the file reference still works, either parsed as markdown, or as plain text.